tests/core/recipes/espresso_recipes/test_espresso.py

```diff
@@ -207,10 +220,17 @@ def test_relax_job_cell(tmp_path, monkeypatch):
         relax_cell=True,
         input_data=input_data,
         pseudopotentials=pseudopotentials,
-        kspacing=0.5,
+        kpts=(1, 1, 1),
```

Comment:
FYI: kpts = (1, 1, 1) leads to gamma with complex wf
kpts = None leads to gamma with real wf (~ 80% faster)

Most of the time your want the later except if you do some special ph.x calc later on or want non-collinear spin